pkg/actions/secure_string.go

```diff
@@ -0,0 +1,39 @@
+package actions
+
+import (
+	"os"
+	"regexp"
+)
+
+// SecureString is a string tha may or may not contain a secret.
+// Secret is considered any value that was read from an environment variable, in the following syntax:
+// "{{ ENV.VAR_KEY }}"
```

Comment:
```suggestion
// SecureString is a string that may be populated from an environment variable.
// If constructed with a string of the form {{ ENV.EXAMPLE_VARIABLE }}, the value is populated from EXAMPLE_VARIABLE and is considered a secret. Otherwise the value is taken from the string as-is, and is not considered a secret.
```